### ISSUE TITLE
Does pretrained ResNet-50 actually use BatchNorm?

### ISSUE THREAD

**Issue by Solomon1588** [zhaoweicai/cascade-rcnn#74]
@zhaoweicai You said "For ResNet, the BatchNorm layers are merged into Scale layers". But I can't find any Batch Norm operation in the Scale Layer? Just in terms of the train.prototxt, it looks like that the training of Resnet-50 doesn't use BN?

**Comment by zhaoweicai** [comment]
"BN merged to Scale layer" means there is no BN layer anymore. Those two layers become a single Scale layer, because at inference BN is just a Scale layer. And that Scale layer is frozen during training. This is a typical strategy for object detection
